.github/actions/feedstock-optional-pr/action.yml

```diff
@@ -11,16 +11,15 @@ runs:
       shell: bash -l {0}
       run: |
         conda install -y conda-build
-    - name: Clone environments
+    - name: Clone build tools
       shell: bash -l {0}
       run: |
-        git clone https://github.com/open-ce/open-ce-environments.git \
-                  ./open-ce-environments
+        git clone https://github.com/open-ce/open-ce-builder.git \
+                  ./open-ce-builder
     - name: Run PR Tests
       shell: bash -l {0}
       run: |
-        test_path="$GITHUB_ACTION_PATH/../../../tests"
-        pip install -e "$GITHUB_ACTION_PATH/../../../"
-        python $test_path/feedstock_tests/check_build_numbers.py \
+        pip install -e open-ce-builer/open_ce
+        python open-ce-builder/tests/feedstock_tests/check_build_numbers.py \
```

Comment:
Do we want to keep the `feedstock_tests` in the builder repo, or should we have that here?

Reply:
I had it here at first, but when linting the file it complained about open-ce libraries not being found. I suppose we could install open-ce before calling pylint, but instead I moved it to the builder repo.

Reply:
With it only being one file, it's not that big a deal. If we ever need to make a PR in the feedstock-test action and in the open-ce-builder repo to fix something, maybe we move it then.